### PR TITLE
Add validation loss plot for Subha

### PR DESCRIPTION
**Pull request by subha-v**
Final validation loss: 3.584375  

I scaled up the model to be wider and deeper (768 hidden dim, 8 layers, 12 heads) compared to the baseline, and increased the context length to 512 to match the validation setting as mentioned by the TAs. I also used torch.compile to speed up training so I could fit more iterations into the 45 minute window. The learning rate was set to 1e-3 with cosine annealing and 200 steps of warmup.

### CODE REVIEW

**Comment by marcelroed** [comment]
Make sure you add an entry to the table as well!

**Comment by subha-v** [comment]
> Make sure you add an entry to the table as well!

Just added! Thank you
